src/test/java/org/symphonyoss/integration/webhook/zapier/parser/ZapierParserResolverTest.java

```diff
@@ -0,0 +1,75 @@
+package org.symphonyoss.integration.webhook.zapier.parser;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.symphonyoss.integration.event.MessageMLVersionUpdatedEventData;
+import org.symphonyoss.integration.model.message.MessageMLVersion;
+import org.symphonyoss.integration.webhook.zapier.parser.v1.V1ZapierParserFactory;
+import org.symphonyoss.integration.webhook.zapier.parser.v1.ZapierPostMessageParser;
+import org.symphonyoss.integration.webhook.zapier.parser.v2.V2ZapierParserFactory;
+import org.symphonyoss.integration.webhook.zapier.parser.v2.V2ZapierPostMessageParser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by crepache on 12/06/17.
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class ZapierParserResolverTest {
+
+  @Spy
+  private List<ZapierParserFactory> factories = new ArrayList<>();
+
+  @InjectMocks
+  private V1ZapierParserFactory v1ParserFactory;
+
+  @InjectMocks
+  private V2ZapierParserFactory v2ParserFactory;
```

Comment:
This field doen't need to be a @@InjectMocks since nothing in this class is being called, just the reference being compared